app/assets/stylesheets/getting-started.scss

```diff
@@ -1,7 +1,11 @@
 #hello-there {
-  .avatar {
-    width: 200px;
-    height: 200px;
```

Comment:
I removed it because it didn't affect the new avatar.

> To have an avatar with the correct size you only need to make sure that this style will only be applied to the uploaded avatar and not to any other avatar on the page.

D'oh! You're right, I've even not thought about it, sorry. I will do.